tests/unit/test_finops.py

```diff
@@ -0,0 +1,34 @@
+import os
```

Comment:
do you plan to add more unit tests for this?

Reply:
no tests for app.py?

Reply:
most of app.py reaches out to S3, so it would require some sort of functional testing framework -- but looking back over it, I might be able to test some of the date processing independent of S3

Reply:
try using the [boto stubber](https://botocore.amazonaws.com/v1/documentation/api/latest/reference/stubber.html), you can find a usage example in [cfn-cr-bucket-tagger](https://github.com/Sage-Bionetworks-IT/cfn-cr-bucket-tagger/blob/master/tests/unit/test_get_bucket_tags.py#L6)

Reply:
cool, thanks, I've updated the tests to use the stubber and some example events